src/scss/pages/_products.scss

```diff
@@ -116,18 +102,14 @@
   display: flex;
   align-items: center;
   gap: 0.25rem;
-  padding-block: 0.5rem;
-  margin-block: 1.25rem;
+  padding: 0.75rem 0.5rem;
+  margin-top: 1.25rem;
```

Comment:
Why the change for non-interpolative naming?

It's better to stay with one padding/margin directive than mix both of those throughout file.